debian/debian/ix-vendor.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+After=middlewared.service
```

Comment:
@yocalebo do we want to run this after we consider the system to be in ready state?

Reply:
Considering we are waiting below for the system to be in ready state - the dependency here should change as well to `ix-postinit`